build.py

```diff
@@ -0,0 +1,41 @@
+import jinja2
+import os
+import shutil
+import subprocess
+import jsonschema
+from ruamel.yaml import YAML
+
+yaml = YAML(typ="safe")
+
+
+def validate():
```

Comment:
Super! Yesterday I was thinking about validating the admin's stuff somehow, glad to see this piece here, that should prevent us/them a lot of errors.